### PR TITLE
BaSP-TG-79: Project Update Form

### PR DESCRIPTION
**Pull request by LucasPrat1**
**Update add and edit form using "React-Hook-Form" library**

- [x] Add schemas with joi
- [x] Add validation with joi. (Work with mode: onChange)
- [x] Set input with previous data for editing
- [x] Do a reset function for cleaning the fields 
- [x] Update the paths from relative paths to absolute paths

### CODE REVIEW

File: src/Components/SuperAdmin/Projects/Form/projectForm.module.css [hunk 1]
```diff
@@ -14,78 +13,52 @@
...
-.container button {
-  width: 100px;
-  padding: 10px 15px;
-  margin-right: 10px;
-  background-color: #771628;
-  border: #4b4b4b solid 1px;
+.container input, select {
+  width: 100%;
+  padding: 5px;
+  border: #373867 solid 2px;
```
Comment (javierbiselli) [inline src/Components/SuperAdmin/Projects/Form/projectForm.module.css:61]: no te aparece el recuadro rojo cuando hay un error de validacion porque sobreescribiste la propiedad

**Review by LucianoAlarcon — APPROVED** [review]
🚀🚀

**Review by javierbiselli — CHANGES_REQUESTED** [review]

**Review by EzeRocco — APPROVED** [review]
good job Lucas!!

File: src/Components/SuperAdmin/Projects/Form/projectForm.module.css (hunk 1 above)
Comment (LucasPrat1) [reply src/Components/SuperAdmin/Projects/Form/projectForm.module.css]: corregido y agregado estilos a los inputs en componentes compartidos!

**Review by javierbiselli — APPROVED** [review]
nice

**Review by martumoser — APPROVED** [review]

**Pull request merged**